requests/altair-aitools-devkit-broken.yml

```diff
@@ -0,0 +1,3 @@
+action: broken
+packages:
+- altair-aitools-devkit/1.0rc1/download/noarch/altair-aitools-devkit-1.0rc1-pyh9fe0891_0.conda
```

Comment:
```suggestion
  - noarch/altair-aitools-devkit-1.0rc1-pyh9fe0891_0.conda
```

Reply:
This was the first version I tried, but for some reason it did not work, and is still not working. Can that be, because the pacakge is *only* published under the rc label and has no prod release yet?

Reply:
@ocefpaf  can you help with this?